android-app-examples/SNetDemo/app/src/main/java/com/example/snetdemo/StyleTransferActivity.java

```diff
@@ -0,0 +1,509 @@
+package com.example.snetdemo;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AlertDialog;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.content.ContextCompat;
+import androidx.core.content.FileProvider;
+
+import android.Manifest;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.content.pm.ActivityInfo;
+import android.content.pm.PackageManager;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.net.Uri;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.os.Environment;
+import android.provider.MediaStore;
+import android.util.Base64;
+import android.util.Log;
+import android.view.View;
+import android.view.WindowManager;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+
+import java.math.BigInteger;
+import java.util.Calendar;
+import java.io.*;
+
+import io.singularitynet.service.styletransfer.StyleTransferOuterClass;
+
+import static com.example.snetdemo.ImageUtils.*;
+
+
+public class StyleTransferActivity extends AppCompatActivity
```

Comment:
Not sure but probably we could move common code from `ImageSegmentationActivity` and `StyleTransferActivity` into base class. What do you think?